src/components/image/notes.stories.mdx

```diff
@@ -0,0 +1,66 @@
+
+import Image from "./"
+import { Meta, Preview, Props, Story } from "@storybook/addon-docs/blocks"
+
+<Meta title="Components/Image" component={Image} />
+
+# Image
+
+The image tag is a utility wrapper that determines how to properly render an image. It also provides the ability to set a background texture (example below).
+
+## Usage
+
+The `src` attribute is what tells the image component how to render the image. See below for examples.
+
+### Gatsby Image
+
+If the `src` is a properly-formatted object, it will be rendered using [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/). This occurs if the object contains a key called `childImageSharp`, which itself contains either a `fluid` or `fixed` object.
+
+<Preview>
+  <Story name="Image">
+    <div style={{ maxWidth: '80%', margin: '10%' }}>
+      <Image src={{
```

Comment:
I had this all ugly and shit because of an issue I ran into with Docz. Do you think there's a way to put a fixture in this directory and import the object we're passing to `src`?